sdk/clientcore/core/src/main/java/io/clientcore/core/implementation/http/rest/SwaggerMethodParser.java

```diff
@@ -148,6 +149,38 @@ public SwaggerMethodParser(Method swaggerMethod) {
             }
         }
 
+        final String[] requestQueryParams = httpRequestInformation.queryParams();
+
+        if (requestQueryParams != null) {
+            for (final String queryParam : requestQueryParams) {
+                final int equalsIndex = queryParam.indexOf("=");
+
+                if (equalsIndex >= 0) {
+                    final String paramName = queryParam.substring(0, equalsIndex).trim();
+
+                    if (!paramName.isEmpty()) {
+                        String paramValue = queryParam.substring(equalsIndex + 1).trim();
+
+                        if (!paramValue.isEmpty()) {
+                            constantQueryParams.compute(paramName, (key, value) -> {
+                                if (value == null) {
```

Comment:
Can simplify this to have this null check if just initialize the value list, that way we only need to write the list add logic once.